api/openapi.yaml

```diff
@@ -425,6 +425,45 @@ paths:
                 type: object
         404:
           description: "Envoy fleet CRD not found"
+  /logs:
+    x-kusk:
+      upstream:
+        service:
+          name: kusk-gateway-api
+          namespace: kusk-system
+          port: 8081
+        rewrite:
+          pattern: "^/api"
+          substitution: ""
+      websocket: true
+    get:
+      tags:
+        - "fleets"
+      summary: "Get envoy fleet logs"
+      operationId: getEnvoyFleetLogs
+      parameters:
+        - name: namespace
+          in: query
+          required: true
+          schema:
+            type: string
+            default: "kusk-system"
+        - name: name
+          in: query
+          required: true
+          schema:
+            type: string
+            default: "kusk-gateway-envoy-fleet"
+      responses:
+        200:
+          description: "Envoy fleet logs"
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
can we change the type to an array of string? because it generates a wrong type

Reply:
What's `it`? Are you using a library on the front end to generate stuff?